addons/website/static/src/common/website_visitor_model.js

```diff
@@ -0,0 +1,35 @@
+import { fields, Record } from "@mail/core/common/record";
+
+export class WebsiteVisitor extends Record {
+    static _name = "website.visitor";
+    static id = "id";
+
+    country = fields.One("res.country", {
+        /** @this {import("models").WebsiteVisitor} */
+        compute() {
+            return this.partner_id?.country ?? this.country_id;
+        },
+    });
+    country_id = fields.One("res.country");
+    /** @type {string} */
+    display_name;
+    displayName = fields.Attr(undefined, {
+        compute() {
+            return (
+                this.partner_id?.name ||
+                this.partner_id?.display_name ||
+                this.display_name ||
+                `Visitor #${this.id}`
```

Comment:
This looks overkill, we should just be returning `name` if that is what we need (or `displayName` from persona).

I don't think we ever send `display_name` on persona again.